index.html

```diff
@@ -70,141 +75,149 @@ <h2 class="yoda">
-          </p>
-          <ul class="grid-list2">
-            <li class="list2">HTML</li>
-            <li class="list2">CSS</li>
-            <li class="list2">Javascript</li>
-          </ul>
-          <h3 class="seeproject">See Project</h3>
+          <img src="Snapshoot Portfolio.svg" alt="" class="gridpicture3" />
+          <div class="grid-container-text">
+            <h2 class="tonic">Facebook 360</h2>
+            <ul class="grid-list">
+              <li class="canopy-list">FACEBOOK</li>
+              <li class="back-list">Full Stack Dev</li>
+              <li class="year-list">2015</li>
+            </ul>
+            <p class="gridtext">
+              Exploring the future of media in Facebook's first Virtual Reality
+              <br />
+              app; a place to discover and enjoy 360 photos and videos on <br />
+              Gear VR.
+            </p>
+            <ul class="grid-list2">
+              <li class="list2">html</li>
+              <li class="list2">Ruby on rails</li>
+              <li class="list2">CSS</li>
+              <li class="list2">Javascript</li>
+            </ul>
+            <h3 class="seeproject">See Project</h3>
+          </div>
         </div>
 
         <div class="grid-container">
-          <img src="Snapshoot Portfolio(3).svg" alt="" class="gridpicture" />
-          <h2 class="tonic">
-            Multi-Post <br />
-            Stories
-          </h2>
-          <ul class="grid-list">
-            <li class="canopy-list">CANOPY</li>
-            <li class="back-list">Back End Dev</li>
-            <li class="year-list">2015</li>
-          </ul>
-          <p class="gridtext">
-            A daily selection of privately <br />
-            personalized reads; no accounts or <br />
-            sign-ups required.
-          </p>
-          <ul class="grid-list2">
-            <li class="list2">HTML</li>
-            <li class="list2">CSS</li>
-            <li class="list2">Javascript</li>
-          </ul>
-          <h3 class="seeproject">See Project</h3>
+          <div class="grid-container-text">
+            <h2 class="tonic">Uber Navigation</h2>
+            <ul class="grid-list">
+              <li class="canopy-list">Uber</li>
+              <li class="back-list">Lead Developer</li>
+              <li class="year-list">2018</li>
+            </ul>
+            <p class="gridtext">
+              A smart assistant to make driving more safe, efficient, and fun
+              <br />
+              by unlocking your most expensive computer: your car.
+            </p>
+            <ul class="grid-list2">
+              <li class="list2">html</li>
+              <li class="list2">Ruby on rails</li>
+              <li class="list2">CSS</li>
+              <li class="list2">Javascript</li>
+            </ul>
+            <h3 class="seeproject">See Project</h3>
+          </div>
+          <img src="Snapshoot Portfolio(3).svg" alt="" class="gridpicture4" />
```

Comment:
- [ ] Kindly try to fix the style here as you see here the image is now down and the content is up in the card in the mobile version, so let;s try to fix the order of the content of the card, so please try to make the image at the top of the card like the other cards and the content in the bottom of it to stick to the Figma, a design 🚀💯
- [ ] Also please try to increase the margin-top of this text see the project as you see here the content is consistent with the above ul, so let's increase the margin-top to stick to the Figma Design 😊💯


> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/CrackedYoda/MyPortfolio/assets/83360636/9e5dcf52-e9cd-4452-bf24-5db126b11b8e)  |  ![image](https://github.com/CrackedYoda/MyPortfolio/assets/83360636/9970256a-6b83-4572-a39c-921bc2a83562) |